src/components/OSCALMarkdownProse.js

```diff
@@ -1,12 +1,23 @@
 import React from "react";
 import ReactMarkdown from "react-markdown";
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+
+const components = {
```

Comment:
`react-markdown` allows for you to change things that come from markdown. 

https://github.com/remarkjs/react-markdown#appendix-b-components

We rewrite the `<a>` tag to display the external link icon.